<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BADAQ Streamer 2.0</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::MediaPlaylistRepeat"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="Message_Viewer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>771</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Btn_Open_Conn">
       <property name="text">
        <string>Open Connection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_Close_Conn">
       <property name="text">
        <string>Close Connection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_Open_file">
       <property name="text">
        <string>Open Raw Data File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_Stream">
       <property name="text">
        <string>Stream PRESQ</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
